src/pages/blog.js

```diff
@@ -1,22 +1,25 @@
 import React from 'react'
+import { graphql } from 'gatsby'
 import BlogLayout from '../layouts/blog'
 
 import '../styles/main.scss'
 
-export default ({data}) => (
+export default ({ data }) => (
   <BlogLayout>
-    <div className="container">
+    {/* <div className="container">
```

Comment:
are we gna remove this commented code?

Reply:
This page has been updated for the blog build out. See PR #247 for further info.